src/multibase.zig

```diff
@@ -513,7 +513,12 @@ pub const MultiBaseCodec = enum {
             }
 
             var dest_index: usize = 0;
-            var num: [1024]u8 align(16) = undefined;
+            var encode_allocator = std.heap.page_allocator;
```

Comment:
It is better to let use choose the allocator, make an allocator arg in func or an allocator field in struct